packages/app/app/components/LibraryView/LibrarySimpleList/index.js

```diff
@@ -23,6 +23,12 @@ const LibrarySimpleList = ({
   t
 }) => (
   <ReactList
+    ref={comp => {
```

Comment:
Does this have to use a ref? Why not use css?

Reply:
Good point, I'll try that. (I basically only use js-inline styles in my projects, so it didn't come to mind at first)

Reply:
There doesn't seem to be any way to "identify" the particular element in question, other than by using the `ref` property. Here is the react-list source: https://github.com/coderiety/react-list/blob/master/react-list.es6

Note that there is no `className` or generic props passthrough for the root container. (which is what we need to style)

I could add the style under `.tracks_container div:nth-child(2)`, except then it would apply the styling to the album and folder-tree views as well.

Reply:
One "hack" is to add an extra container around the react-list component, with a unique class. Then I could add the styling for `.library_simple_list_container > div`.

Should I do that, or stick with the ref approach?

Reply:
Ok, that's too bad that the library doesn't expose a way to get to that container.

You could import the styles from the album view and tree view and use the `:not` selector but that's too much code for little benefit. Instead of directly setting the `display` property using the ref, could you please set a class on the ref and use that class in scss? This way if we want to apply more styling to that component in the future it will be easier.

Reply:
Done.